sequential_parallel.py

```diff
@@ -0,0 +1,15 @@
+from threading import Thread
+
+
+def start_parallel(functions, arguments):
```

Comment:
I think it is cleaner to pack each function with it's arguments within single data type.
Check out dataclass - it can be useful.

see for example the following code:
1:
    start_parallel((do_work1, do_work2, do_work3), ((a, b, c), (a, 1) ,())

2:
    job1 = Job(do_work1, (a, b, c))
    job2 = Job(do_work2, (a, 1)
    job3 = Job(do_work3, ())
    start_parallel(job1, job2, job3)

3:
    start_parallel(Job(do_work1, (a, b, c)), Job(do_work2, (a, 1)), Job(do_work3, ()))

I think 2 or even 3 are much better than 1

Reply:
Good idea, thank you.
**Done!**